src/index.ts

```diff
@@ -53,7 +53,8 @@ export type TranslateGlobal<
 > = TranslateType<L, TG>;
 
 export type Interpolation =
-  | ({ smart_count?: number } & {
-      [key: string]: string | object;
-    })
+  | {
```

Comment:
isn't this possible to type like this?

```ts
type Interpolation = {
    smart_count?: number;
    [key: string]: string | number | object | undefined; 
};
```

https://www.typescriptlang.org/play?#code/C4TwDgpgBAkgdsCAnMB7ANgQ2AS1XKAXigG8AoKSqAZwFtMlgB9AY1QFcEB+ALijna0ARsgDcFKgG0A1hBB9qwJDjgBzALoKlK1VAA+-QSKT6oqIQCsILYKc4ATCADMVEe6KhkAvqKA

Reply:
you are right that is better

Reply:
@davidfurlong yes but numbers in translations should be allowed, right?

Reply:
Improved, thanks